hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashOzoneFileSystem.java

```diff
@@ -386,6 +393,7 @@ boolean processKeyPath(List<String> keyPathList) {
         OzoneManagerProtocolProtos.OMRequest omRequest =
             getRenameKeyRequest(src, dst);
         try {
+          ozoneManager.getMetrics().incNumTrashFilesRenames();
```

Comment:
Out of curiosity: what is the difference between `incNumTrashFilesRenames` and ` incNumTrashRenames`, does the latter contains the former or vice verse?

Reply:
`numTrashRenames` would count the renames on the directory while `numTrashFileRenames` would count renames on each file of the directory.

Reply:
Thanks for the clarification!